deploy/group_vars/all.yml

```diff
@@ -106,11 +106,11 @@ k8s_aws_load_balancer_type: nlb
 # caktus.k8s-hosting-services: Logging and monitoring configuration
 # ----------------------------------------------------------------------------
 
-k8s_papertrail_logspout_destination: "syslog+tls://logs2.papertrailapp.com:20851"
-k8s_papertrail_logspout_memory_limit: 128Mi
+# k8s_papertrail_logspout_destination: "syslog+tls://logs2.papertrailapp.com:20851"
```

Comment:
I’m pretty certain this isn’t needed anymore since we moved away from Papertrail, but I did not want to delete without asking first.

Reply:
@ronardcaktus I would like to defer to either @copelco or @tobiasmcnulty on removing this from the codebase.